src/shared/components/mutationMapper/MutationMapperDataStore.ts

```diff
@@ -196,6 +196,19 @@ export default class MutationMapperDataStore
         return countDuplicateMutations(this.tableDataGroupedByPatients);
     }
 
+    @computed
+    get tableDataSamples() {
+        return _.uniqBy(_.flatten(this.tableData), 'sampleId').map(m => ({
```

Comment:
property shorthands are not recommended because they are not type safe. this is better:

```suggestion
        return _.uniqBy(_.flatten(this.tableData), m => m.sampleId).map(m => ({
```